design/decisions/why-sql/index.qmd

```diff
@@ -0,0 +1,122 @@
+---
+title: Why PostgreSQL
+description: |
+  SQL is the backbone of any database system. However there are many variants
+  of SQL. PostgreSQL is a powerful and feature-full variant.
+author: Kristiane Beicher
+date: last-modified
+categories:
+  - backend
+  - database
+  - software
+  - tools
+  - technology
+  - software-architecture
+---
+
+## Context and Problem Statement
+
+To fit our [Guiding Principles](/design/software-architecture/introduction.qmd#guiding-principles) of using established and well-known technologies, we're only going to look at the top three open source relational databases (as defined by [DB-Engines](https://db-engines.com/en/ranking/relational+dbms) in November 2022): MySQL, PostgreSQL, and SQLite.
+
+We have chosen to focus on relational databases, as we believe that the user base for a project like Seedcase will, overall, be more familiar with this form for database system. Moreover, we will have users that will require access to the raw data and therefore we are not looking at systems like MongoDB and Redis.
```

Comment:
Add links to MongoDB and Redis? Why can't users access raw data with these?

Reply:
Don't actually know where I picked that one up, but I've rephrased it.